Server/ConcordiaCurriculumManager/Services/CourseService.cs

```diff
@@ -174,5 +175,24 @@ public async Task<CourseModificationRequest> InitiateCourseModification(CourseMo
         _logger.LogInformation($"Created ${typeof(CourseCreationRequest)} ${courseModificationRequest.Id}");
 
         return courseModificationRequest;
+    }
+
+    public async Task<Course?> GetCourseData(CourseDTO c)

```

Comment:
Logic here is fine for now, but we'll almost certainly be changing it next sprint as we iron out some of our versioning kinks.

Reply:
Sounds good